src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public enum Colors {
+    Blue,

```

Comment:
```suggestion
    BLUE,
```
same for others - enum values are constants